src/developer/web-api/tracker.md

```diff
@@ -2130,6 +2130,66 @@ An example of a json response:
 The response will be the same as the collection endpoint but referring to a single tracked
 entity, although it might have multiple rows for each attribute.
 
+#### Tracked entity attribute value change logs `GET /api/tracker/trackedEntities/{uid}/changeLogs`
+
+This endpoint retrieves change logs for the attributes of a specific tracked entity. It returns a list of all tracked entity attributes that have changed over time for that entity.
```

Comment:
Maybe shorten to

This endpoint retrieves change logs for a tracked entity's attribute values, showing all changes over time.